apps/desktop/src/styles.css

```
@@ -274,6 +274,18 @@ html.dark {
 }
 
+/* Clip the root element to the window's overlay radius, preventing
+   WebView2's rectangular dark edge from showing at the corners.
+   Reset to 0 when the window is maximized to avoid rounding the
+   OS rectangular frame. */
+html {
+  clip-path: inset(0 round var(--zephyr-radius-overlay, 24px));
+}
+html.is-maximized {
+  --zephyr-radius-overlay: 0px;
+  clip-path: none;
```

Stylelint (17.14.1)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)